### PR TITLE
Add option to be quiet

### PR DESCRIPTION
**Pull request by michaelmwu**
it isn't possible to replicate `--quiet` with just cliEngineConfig. Add that option in the eslint-output cli options.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -8,7 +8,10 @@ const debug = require('debug')('eslint-output');
 const rc = require('./config');
 
 const cwd = path.resolve(process.cwd());
-const { maxWarnings } = yargs.argv;
+const { maxWarnings, quiet } = yargs.options({
+  'maxWarnings': { type: 'number', default: false },
```
Comment (lwhiteley) [inline app.js]: fix eslint warnings for these

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-output",
-  "version": "1.2.2",
+  "version": "1.2.3",
```
Comment (lwhiteley) [inline package.json]: can undo this change. will bump the version for the release using the `release:patch` script

File: app.js [hunk 3]
```diff
@@ -23,6 +26,10 @@ const cli = new CLIEngine(
 
 const report = cli.executeOnFiles(rc.files || ['.']);
 
+if (quiet) {
+  report.results = CLIEngine.getErrorResults(report.results);
```
Comment (michaelmwu) [inline app.js]: this came from here: https://github.com/eslint/eslint/blob/219aecb78bc646d44bad27dc775a9b3d3dc58232/lib/cli.js#L216

**Comment by lwhiteley** [comment]
v1.2.3 released to npm
